### PR TITLE
Caleb/cleanup

### PR DESCRIPTION
**Pull request by komali2**
Found some cruff

### CODE REVIEW

**Comment by komali2** [comment]
Fixes https://github.com/trynmaps/opentransit-map/issues/59

**Comment by ckrajewski** [comment]
@komali2 we should also remove this from yarn.lock file (running uninstall)

**Comment by komali2** [comment]
this lol

https://github.com/relayjs/relay-examples/issues/58

**Comment by komali2** [comment]
@ckrajewski right goo dpoint

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Check out our welcome doc: <http://bit.ly/opentransit-onboarding>.
 
 **Web app**
 
-* Tryn-React (https://github.com/trynmaps/tryn-react): React, Relay, Flow
+* Opentransit-Map https://github.com/trynmaps/opentransit-map): React, Relay, Flow
```
Comment (ckrajewski) [inline README.md]: Someone actually updating the Readme
I'm impressed haha

**Comment by ckrajewski** [comment]
Looks good to me 👍

**Comment by ckrajewski** [comment]
@komali2 
just assign yourself to this issue, and close when merged
https://github.com/trynmaps/opentransit-map/issues/59

**Pull request merged**